test/ftpserver_test.rb

```diff
@@ -4,6 +4,16 @@
 
 require "yast"
 
+# stub module to prevent its Import
+# Useful for modules from different yast packages, to avoid build dependencies
+def stub_module(name)
+  Yast.const_set name.to_sym, Class.new { def self.fake_method; end }
+end
+
+stub_module("Users")
+
+
```

Comment:
I would like to do it as separate pull request as it is out of scope this PBI and I would like to focus on critical path reduction.